be/src/Project/Services/News/News.Api/Services/News/NewsService.cs

```diff
@@ -70,6 +70,17 @@ public async Task<Guid> CreateAsync(CreateNewsItemServiceModel model)
                 await this.newsContext.NewsItemFiles.AddAsync(file.FillCommonProperties());
             }
 
+            foreach (var groupId in model.GroupIds.OrEmptyIfNull())
```

Comment:
**suggestion:** please replace group with clientGroup everywhere. clientGroupId instead of groupId, ClientGroupIds instead of GroupIds, etc.